scripts/validationAndPreparation.sh

```diff
@@ -1,45 +1,64 @@
+#!/bin/bash
```

Comment:
To be totally honest, I have a very hard time understanding what's going on here. I know this is not really actionable feedback, but maybe you can spend some time refactoring this code into a slightly more readable format.

Maybe some common commands can be aliased, or some functionality pulled into their own functions.

Reply:
sorry.

I have never ever written a longer bash file before. I do not know all the conventions etc. 
Also it took me forever to get all these long and stupid commands together...